EliteAPI/Discord/RichPresenceClient.cs

```diff
@@ -235,14 +235,26 @@ private void DoAutomaticEvents()
                     }
                 }
             };
-            if (string.IsNullOrWhiteSpace(api.Location.StarSystem))
-            {
-                UpdatePresence(new RichPresence { Text = "Just started playing", Icon = "ed", IconText = "EliteAPI" });
-            }
-            else
-            {
-                UpdatePresence(new RichPresence { Text = "Just started playing", TextTwo = "In " + api.Location.StarSystem, Icon = "ed", IconText = "EliteAPI" });
-            }
+
+            api.Events.StatusIsRunning += (sender, e) => {
+                if (api.Status.IsRunning)
+                {
+                    rpc.ClearPresence();
+                    return;
+                }
+
+                if (string.IsNullOrWhiteSpace(api.Location.StarSystem))
+                {
+                    UpdatePresence(new RichPresence { Text = $"Just started playing", Icon = "ed", IconText = "EliteAPI" });
+                }
+                else
+                {
+                    UpdatePresence(new RichPresence { Text = $"Just started playing", TextTwo = "In " + api.Location.StarSystem, Icon = "ed", IconText = "EliteAPI" });
+                }
+            };
+            // Fire it off if it is running already
+            if (api.Status.IsRunning) 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/EliteAPI/EliteAPI/pullRequest?prid=5331244)